shared/src/commonMain/kotlin/data/credentials/EuPidCredentialAdapter.kt

```diff
@@ -130,7 +134,7 @@ sealed class EuPidCredentialAdapter(
                 ADDRESS_HOUSE_NUMBER -> Attribute.fromValue(residentHouseNumber)
                 ADDRESS_COUNTRY -> Attribute.fromValue(residentCountry)
                 ADDRESS_REGION -> Attribute.fromValue(residentState)
-                GENDER -> Attribute.fromValue(gender)
+                SEX -> Attribute.fromValue(sex)
```

Comment:
so they once understood the difference between sex and gender and forgot about it now 🤦